<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionarLocal</class>
 <widget class="QDialog" name="gestionarLocal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionar Locales</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="buscarLocalTxt"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Buscar Local: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buscarPb">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editarLocalPb">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>500</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registrarLocalPb">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>240</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Registrar Local</string>
   </property>
  </widget>
  <widget class="QPushButton" name="habilitarInLocalPb">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>160</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Habilitar/Inhabilitar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regresarPb">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>320</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twMostrarLocales">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>741</width>
     <height>331</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Código</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dirección</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Estado</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDireccion">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>510</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>470</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dirección</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buscarLocalTxt</tabstop>
  <tabstop>buscarPb</tabstop>
  <tabstop>editarLocalPb</tabstop>
  <tabstop>registrarLocalPb</tabstop>
  <tabstop>habilitarInLocalPb</tabstop>
  <tabstop>regresarPb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
